File: TypeScript/1005.ts
var inputs = require('fs').readFileSync('/dev/stdin', 'utf8').split('\n');

var a = Number(inputs.shift());
var b = Number(inputs.shift());
   
console.log(`MEDIA = ${(((a * 3.5 + b * 7.5) / 11).toFixed(5))}`);

File: 1. Beginner/1005/1005.scala
object Main {
    def main(args: Array[String]) {
        val a = scala.io.StdIn.readDouble()
        val b = scala.io.StdIn.readDouble()
        val r = (a * 3.5 + b * 7.5) / 11
        println(f"MEDIA = $r%.5f")
    }
}